src/EnergyPlus/Psychrometrics.cc

```diff
@@ -570,10 +573,9 @@ namespace Psychrometrics {
             WBT = ResultX;
 
             // If converged, leave iteration loop.
-            if (icvg == 1) break;
-
-            // Error Trap for the Discontinuous nature of PsyPsatFnTemp function (Sat Press Curve) at ~0 Deg C.
-            if ((PSatstar > 611.000) && (PSatstar < 611.25) && (std::abs(error) <= 0.0001) && (iter > 4)) break;
+            if (icvg == 1) {
+                break;
+            }
```

Comment:
Removed the "Error Trap for the Discontinuous nature of PsyPsatFnTemp function (Sat Press Curve) at ~0 Deg C." since it's no longer discontinuous.

Reply:
I vaguely remember this. It looks like the unit tests hits this so good there. Gotta check it though.

Reply:
Yeah, I remember this now, this is a check for where this used to have a problem. Now that the problem is corrected, this check is no longer needed.